rules/gcp_audit_rules/gcp_iam_service_accounts_get_access_token_privilege_escalation.py

```diff
@@ -0,0 +1,20 @@
+from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_walk
+
+
+def rule(event):
+    authorization_info = deep_walk(event, "protoPayload", "authorizationInfo")
+    if not authorization_info:
+        return False
+
+    for auth in authorization_info:
+        if (
+            auth.get("permission") == "iam.serviceAccounts.getAccessToken"
+            and auth.get("granted") is True
+        ):
+            return True
+    return False
+
```

Comment:
Please add a title function similar to https://github.com/panther-labs/panther-analysis/blob/release/rules/gcp_audit_rules/gcp_privilege_escalation_by_deployments_create.py#L16